### ISSUE TITLE
Autostartup without asking admin rights

### ISSUE THREAD

**Issue by Den4enko** [kellwinr/galaxybook_mask#13]
Is this possible to autostart this script with admin privileges without asking for them?

**Comment by kellwinr** [comment]
I don't think this is possible.

**Comment by Den4enko** [comment]
> I don't think this is possible.

I did this by adding this script to the Windows Task Scheduler

**Comment by Den4enko** [comment]
But it's complicated

**Comment by kellwinr** [comment]
yeah, I don't want to complicate things, you can do that on your own, but it takes time manually.

**Comment by rxunique** [comment]
> I did this by adding this script to the Windows Task Scheduler

How did you do it? I had it in Task Schedule on logged on with highest privileges checked, still get windows admin pop up
